### PR TITLE
Listings Redone, Added Name, Applicants Redone

### PR DESCRIPTION
**Pull request by JasonZuang**
Created a Service for Applicants and Listings (Tested)
Added Name to SQL Script and Listing model (Tested)
Added applicants/delete/ to jwt webconfig

### CODE REVIEW

File: src/main/resources/application.properties [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # change this to dev, but do not commit this file
 # also change the password and username of your DB defined in application-dev.properties file
-spring.profiles.active=prod
+spring.profiles.active=dev
```
Comment (Zhen-Guan) [inline src/main/resources/application.properties:3]: please do not commit this file

File: src/main/resources/scripts/createDB.sql [hunk 2]
```diff
@@ -26,6 +26,7 @@ CREATE TABLE IF NOT EXISTS landlord (
 CREATE TABLE IF NOT EXISTS listing (
   id INT PRIMARY KEY AUTO_INCREMENT,
   landlord_email VARCHAR(255) NOT NULL,
+  name TEXT NOT NULL,
```
Comment (Zhen-Guan) [inline src/main/resources/scripts/createDB.sql]: please use VARCHAR(255) here because name is supposed to be short.

**Comment by Zhen-Guan** [comment]
LGTM overall, please make changes based on the review.

**Comment by JasonZuang** [comment]
K

File: .gitignore [hunk 3]
```diff
@@ -31,3 +31,6 @@ build/
 
 ### VS Code ###
 .vscode/
+
+#Prop
+.properties
```
Comment (Zhen-Guan) [inline .gitignore]: Can you only add the application.properties file in .gitignore? This will disallow us to commit the other two properties files.

**Review by Zhen-Guan — APPROVED** [review]

**Pull request merged**